Portfolio/index.css

```diff
@@ -1,174 +1,126 @@
-console.log('Hello World');

```

Comment:
The wildcard selector (*) is not recommended as it can have unintended side effects. It is better to explicitly select the elements you want to style.